worksheet_A/TerminalHacking.cpp

```diff
@@ -41,6 +45,66 @@ int main()
 
 	// TODO: implement the rest of the game
 
+	// initialise variables
+	bool WordInList = false;
+	std::cout << secret;
+	std::string guess;
+	int lives = 4;
+	std::string result = "Lost";
+
+	// main game loop
+	while (lives > 0) 
+	{
+
+		// Ask for user guess
+		std::cout << " what is your guess?" << std::endl;
+		std::cin >> guess;
+
+		// Check to see if guess is in word set
+
+
+
+		for each (std::string word in options)
```

Comment:
There is a built-in (and more efficient) way of checking whether a `std::set` contains an element -- see http://stackoverflow.com/questions/1701067/how-to-check-that-an-element-is-in-a-stdset